exercises/execise-1.js

```diff
@@ -8,10 +8,16 @@
   
 */
 
-function average() {
-  // TODO: implement
+function average(num1 = 0, counter = 0) {
```

Comment:
This is working okay but a good practice would be save this values into variables and not params, in all cases avoid to modify the value of params directly, if you have to do it, make sure to copy it.

For instance: 

```javscript
  function (param1, param2) {
     let param1ToBeModified = param1;
  }
```